recipes-rubygems/rubygems-ffi-libarchive_1.0.17.bb

```diff
@@ -6,11 +6,11 @@ HOMEPAGE = "https://github.com/chef/ffi-libarchive"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=8f7bb094c7232b058c7e9f2e431f389c"
 
-EXTRA_DEPENDS += "\
+EXTRA_DEPENDS_append += " \
     libarchive \
 "
 
-EXTRA_RDEPENDS += "\
```

Comment:
potential issue: Use '_append' instead of ' += ' as it overwrites 'EXTRA_DEPENDS ?= ""